src/views/main/Home.vue

```diff
@@ -1,37 +1,27 @@
 <template>
   <v-container :class="{ 'pa-1': !$vuetify.breakpoint.mdAndUp }" fluid>
-    <v-overlay v-model="overlay" opacity="0.5" z-index="10">
-      <v-card elevation="2" rounded v-show="showUserAgreement" color="white">
-        <v-card-title class="primary--text">{{ $t('使用前必读') }}</v-card-title>
-        <div class="black--text ma-4">
-          请仔细阅读<a class="text-decoration-none" href="https://about.cha.fan/docs"
-            >本网站相关文档</a
-          >。 如果你继续使用本网站，将视为同意「网站用户协议」并了解「社区行为守则」的内容。
-        </div>
-        <v-card-actions class="ma-2">
-          <v-spacer />
-          <v-btn color="primary" @click="continueUserAgreement">同意</v-btn>
-        </v-card-actions>
-      </v-card>
-      <v-card color="white" elevation="2" rounded v-show="showFabHint">
-        <v-card-title class="primary--text">{{ $t('Try the explore button') }} ↘</v-card-title>
-      </v-card>
-    </v-overlay>
-    <v-row justify="center" class="pa-3">
+    <user-agreement
+      :overlay="overlay"
+      :show-fab-hint="showFabHint"
```

Comment:
I thought the prop name might match the exact variable name? i.e.g `:showFabHint="showFabHint"` (it might be a limitation of using decorator to define prop, but it looks buggy now in vercel preview).

Reply:
I saw the production build bug, tring fix it.